docs/getting_started.md

```diff
@@ -300,19 +284,33 @@ config. To learn more on how to create a container with AML SDK click
 [here](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.image.image.image?view=azure-ml-py#create-workspace--name--models--image-config-).
 
 Below is release pipeline with two tasks one to create an image using the above
-script and second is the deploy the image to Web App for containers
-![release_webapp](./images/release-webapp-pipeline.PNG).
+script and second is the deploy the image to Web App for containers.
+
+![release_webapp](./images/release-webapp-pipeline.PNG)
+
+In the Variables tab, link the pipeline to your variable group (`devopsforai-aml-vg`). In the variable group definition, add the following variables:

```

Comment:
These should already be picked up by the variables template file.  If you want to add instructions for people to modify that file, then no problem else remove this section.

Reply:
This is the doc for the release pipeline, which if I'm not mistaken the user must create by hand from scratch. (This is "resolved" in PR #122 that replace the release pipeline with a multi-stage build pipeline)